lms/static/js/learner_dashboard/models/program_model.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Model for Course Programs.
+ */
+(function (define) {
+    'use strict';
+    define(['backbone'], function (Backbone) {
+        var Program = Backbone.Model.extend({
```

Comment:
No need to assign to a var and then return the var, you can just return Backbone.Model.extend({....